src/pages/saksbehandling/revurdering/bosituasjon/BosituasjonFormPage.tsx

```diff
@@ -75,7 +75,9 @@ const BosituasjonFormPage = (props: RevurderingStegProps) => {
                                 <BooleanRadioGroup
                                     legend={formatMessage('form.harSøkerEPS')}
                                     error={fieldState.error?.message}
-                                    {...field}
+                                    value={field.value}
+                                    onChange={field.onChange}
+                                    name={field.name}
```

Comment:
Hva slags feilmelding er det disse skal løse opp i? Blir det ikke det samme bare at man ikke hekter opp `ref` og evt. `onBlur`?

Reply:
Får feilmelding i forbindelse med `ref`
`BooleanRadioGroup: `ref` is not a prop.` Noe som i og for seg er litt rart, ettersom ref jo er et prop 🤷

Reply:
Det hørtes jo veldig merkelig ut 🤔

Reply:
03cb21ec3b029813c8f23d79bb488f97c1895ed5